spec/requests/widgets_spec.rb

```diff
@@ -62,43 +65,66 @@
         expect(response).to have_http_status(:not_found)
       end
 
-      it "generate_content of a single Widget" do
-        miq_widget = FactoryBot.create(:miq_widget)
-
-        api_basic_authorize('widget_generate_content')
-
-        expect(MiqTask.count).to eq(0)
-        post(api_widget_url(nil, miq_widget), :params => gen_request(:generate_content))
-
-        expect_single_action_result(:success => true, :message => /#{miq_widget.id}.* content generation/i, :href => api_widget_url(nil, miq_widget))
-        expect(MiqTask.first.message).to match(/content generation\]\ being run for user/)
+      context "generate_content for a group" do
+        before do
+          stub_settings(::Settings.to_hash.merge(:product => {:report_sync => true}))
+          MiqReport.seed_report("Vendor and Guest OS")
+          feature1 = MiqProductFeature.find_all_by_identifier("dashboard_admin")
+          @user1   = FactoryBot.create(:user, :role => "role1", :features => feature1)
+          @group1  = @user1.current_group
+          @ws = FactoryBot.create(:miq_widget_set, :name => "Home", :userid => @user1.userid, :group_id => @group1.id)
+          miq_widget.make_memberof(@ws)
+          api_basic_authorize('widget_generate_content')
+        end
+
+        it "single widget" do
```

Comment:
can you create a sentence what it does ? 

for example:
`it "generates single widgets "`